pages/shields/oled.rst

```diff
@@ -0,0 +1,139 @@
+OLED Shield
+***********
+
+.. highlight:: python
+   :linenothreshold: 4
+
+.. figure:: /images/oled_shield_top.png
+   :width: 270
+
+   OLED Shield
+
+The `OLED Shield`_ is a Wemos D1 Mini shield that hosts a *tiny* 17mm 64x48
+monochrome OLED display.
+
+.. NOTE::
+
+    What is an *OLED*? It stand for Organic Light-Emitting Diode. It describes
+    the technology used to turn on or off pixels - dots - on the display. The
+    term is largely unimportant, just think of it like a tiny little TV where
+    you can control every pixel.
+
+Although small, this shield packs a punch! You can draw pixels, render lines
+and display text.
+
+The *integrated circuit* that makes the magic happen is an `SSD1306`_. It sits
+between the microcontroller and the raw display and allows us to send commands
+to make stuff appear. Because the SSD1306 is a common chip there is a driver
+for it built-in to MicroPython.
```

Comment:
>Because the SSD1306 is a common chip there is a driver for it built-in to MicroPython.
The implication is that MicroPython has drivers for all common chips (definitely not true), may want to consider rewording

Reply:
Fixed in 9e2e99c